src/control/cmd/daos/filesystem.go

```diff
@@ -428,3 +429,49 @@ func (cmd *fsFixRootCmd) Execute(_ []string) error {
 
 	return nil
 }
+
+type fsDfuseQueryCmd struct {
+	daosCmd
+
+	Path string `long:"path" description:"Path to evict from dfuse" required:"1"`
```

Comment:
The command is `memquery`, but this says `Path to evict`?

Reply:
Will update.